### ISSUE TITLE
Panic when checking empty input

### ISSUE THREAD

**Issue by amake** [crate-ci/committed#192]
I am on macOS and I find that committed panics on empty input:

```sh
$ pwd
/path/to/my/git/repo
$ echo '' | RUST_BACKTRACE=full ~/Downloads/committed-v0.1.25-x86_64-apple-darwin/committed --commit-file -
thread 'main' panicked at 'Subject should have at least one word', src/checks.rs:219:10
stack backtrace:
   0:        0x1093d2a24 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h5b5b7de1c79b4b45
   1:        0x1093eff3e - core::fmt::write::h65ec8c8a6aa7b549
   2:        0x1093d066a - std::io::Write::write_fmt::hc52e560ab9e57db4
   3:        0x1093d477f - std::panicking::default_hook::{{closure}}::h0974624c30a6e2b5
   4:        0x1093d4289 - std::panicking::default_hook::hda3cfa32e0773aac
   5:        0x1093d4de5 - std::panicking::rust_panic_with_hook::h1cdaa4b48d8ea4a1
   6:        0x1093d4925 - std::panicking::begin_panic_handler::{{closure}}::h51157b4dbc4c5d91
   7:        0x1093d2eb8 - std::sys_common::backtrace::__rust_end_short_backtrace::h8f7fee8b4c5a0c05
   8:        0x1093d488a - _rust_begin_unwind
   9:        0x1093ffe0f - core::panicking::panic_fmt::h28a37220b8fae52a
  10:        0x1093ffd1a - core::option::expect_failed::ha8e5b47364c1609f
  11:        0x1092278c6 - committed::checks::check_message::h97d4a42351e93287
  12:        0x10922fbe6 - committed::run::h8952e2bcead92478
  13:        0x109230989 - committed::main::h22e64243df582690
  14:        0x109227c3a - std::sys_common::backtrace::__rust_begin_short_backtrace::h9af9bf8506edc263
  15:        0x10922928c - std::rt::lang_start::{{closure}}::h39ac97f7ce46975f
  16:        0x1093d5231 - std::rt::lang_start_internal::h37583e1fa93097c4
  17:        0x109230e69 - _main
```

I get similar results for referencing a file rather than stdin:

```sh
$ touch foo  
$ RUST_BACKTRACE=full ~/Downloads/committed-v0.1.25-x86_64-apple-darwin/committed --commit-file foo    
thread 'main' panicked at 'Subject should have at least one word', src/checks.rs:219:10
stack backtrace:
⋮
```

I also see a panic when executing outside of a repository. This might be intentional, but I thought it would be nice to allow execution from anywhere if supplying the message directly (as opposed to reading it from the repository).

```sh
$ cd
$ echo '' | RUST_BACKTRACE=full ~/Downloads/committed-v0.1.25-x86_64-apple-darwin/committed --commit-file -
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: could not find repository from '/Users/amake'; class=Repository (6); code=NotFound (-3)', src/main.rs:225:22
stack backtrace:
   0:        0x1067c7a24 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h5b5b7de1c79b4b45
   1:        0x1067e4f3e - core::fmt::write::h65ec8c8a6aa7b549
   2:        0x1067c566a - std::io::Write::write_fmt::hc52e560ab9e57db4
   3:        0x1067c977f - std::panicking::default_hook::{{closure}}::h0974624c30a6e2b5
   4:        0x1067c9289 - std::panicking::default_hook::hda3cfa32e0773aac
   5:        0x1067c9de5 - std::panicking::rust_panic_with_hook::h1cdaa4b48d8ea4a1
   6:        0x1067c9925 - std::panicking::begin_panic_handler::{{closure}}::h51157b4dbc4c5d91
   7:        0x1067c7eb8 - std::sys_common::backtrace::__rust_end_short_backtrace::h8f7fee8b4c5a0c05
   8:        0x1067c988a - _rust_begin_unwind
   9:        0x1067f4e0f - core::panicking::panic_fmt::h28a37220b8fae52a
  10:        0x1067f4ef5 - core::result::unwrap_failed::ha7403d21b7149ada
  11:        0x106622a3e - core::result::Result<T,E>::unwrap::h6a56d71af627d3b2
  12:        0x106625994 - committed::main::h22e64243df582690
  13:        0x10661cc3a - std::sys_common::backtrace::__rust_begin_short_backtrace::h9af9bf8506edc263
  14:        0x10661e28c - std::rt::lang_start::{{closure}}::h39ac97f7ce46975f
  15:        0x1067ca231 - std::rt::lang_start_internal::h37583e1fa93097c4
  16:        0x106625e69 - _main
```

**Comment by epage** [comment]
Fixed in #193 and now released.

**Comment by amake** [comment]
Thanks!
